recipe/run_test.bat

```diff
@@ -1,7 +1,10 @@
 @echo on
 
-"%PYTHON%" -c "import dpctl"
+"%PYTHON%" -c "import dpctl; print(dpctl.__version__)"
 if errorlevel 1 exit 1
 
-python -m pytest -q -ra --disable-warnings --pyargs dpctl -vv
+"%PYTHON%" -c "import dpctl; dpctl.lsplatform()"
 if errorlevel 1 exit 1
+
+python -m pytest -q -p no:faulthandler -ra --disable-warnings --pyargs dpctl -vv
```

Comment:
why do we use here suddenly 'build'-sections python?  as above we actually are using the variant from 'host' (%PYTHON% is the host one)